src/app/(app)/members/components/MemberFormBuilder.tsx

```diff
@@ -103,10 +103,19 @@ export const userBuilderSchema = z.object({
       },
     }
   ),
-  ehad_duration: z
-    .date()
-    .min(addDays(new Date(), 30), "Minmum Ehad Duration is one Month"),
-});
+  ehad_duration: z.date().refine(
+    (date) => {
+      const today = new Date();
+      const minDate = new Date();
+      minDate.setDate(today.getDate() - 30);
+      return date <= minDate; 
+    },
+    {
+      message: "Ehad Duration must be more than 30 days ago",
```

Comment:
@ghulamhafeez `ago`  is used for past dates

Reply:
Yes, I know😅 you have a misconception. This field is designed to select past days, not future ones